src/PlayableCharacter.java

```diff
@@ -0,0 +1,71 @@
+
+public abstract class PlayableCharacter extends Character{
+
+	//Atributes
+	private static byte nextlevelXP = 100; 	//Used for levelingUp
+	private int experience;					//Current XP, can't exceed experience for next level.
+	private Weapon equipedWeapon;
+	private Armor equipedArmor;
```

Comment:
We're not going to use items (for now), so this 2 attributes should be commented or removed